src/tasks/mixin/battle_mixin.py

```
@@ -426,32 +426,40 @@ def auto_battle(self, no_battle: bool = False):
         sleep_time = self.get_battle_config("进入战斗后的初始等待时间", 3)
 
         while True:
 
             # 全局超时保护
-            if self.active_time() - start_time > 420:
+            if self.active_time() >= deadline:
                 self.log_info("自动战斗超时")
                 return False
 
-            # 战斗结束判定
+            # 战斗结束后轮询结算状态，成功路径不再固定等待 5 秒。
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)